sinks/api/supported_metrics.go

```diff
@@ -255,7 +256,50 @@ var statMetrics = []SupportedStatMetric{
 		},
 		OnlyExportIfChanged: true,
 	},
-
+	{
+		MetricDescriptor: MetricDescriptor{
+			Name:        "cpu/node_usage",
+			Description: "Cumulative CPU usage on all cores on a node",
+			Type:        MetricCumulative,
+			ValueType:   ValueInt64,
+			Units:       UnitsNanoseconds,
+		},
+		HasValue: func(spec *source_api.ContainerSpec) bool {
+			return spec.HasCpu && spec.HasResourceId
```

Comment:
We don't want to use regular `cpu/usage` metric, as it contains info from both machines and containers. We want our metric to contain info for **machines only**.

I agree that annotating `ContainerSpec` with `isNode` is better idea than basing on `resourceId` to be not empty. We will apply it.